draft-jasdips-regext-rdap-rpki.md

```diff
@@ -214,8 +216,6 @@ Syntax: rpki1/roas?name=<name search pattern>
 
 Syntax: rpki1/roas?startAddress=<IP address>&prefixLength=<CIDR length>
 
-A> Can this be rpki1/roas/startAddress/length as in rpki1/roas/192.16.0.0/16?
```

Comment:
There is also "A> See note above." later.

Reply:
let me fix